apps/mobile/src/libs/components/background-wrapper/libs/maps/preset-to-planet-position.map.ts

```diff
@@ -0,0 +1,106 @@
+import {
+	type colorToGradientColors,
+	type directionToGradient,
+	type sizeToStyles,
+} from "~/libs/maps/maps";
+import { type StyleProp, type ViewStyle } from "~/libs/types/types";
+
+type Preset = "default" | "signIn" | "signUp" | "welcome" | "wheelLoading";
```

Comment:
It can be an option, but it brings unnecessary complexity. We won't use these preset values for other components. As we are using this specifically as a preset prop for one exact component we are good with union type here.